### PR TITLE
rendered iframes now allow autoplay and fullscreen

### PR DESCRIPTION
**Pull request by supermuesli**
Fixes bug https://github.com/sarbagyastha/youtube_player_flutter/issues/582

### CODE REVIEW

**Comment by adamsmaka** [comment]
doesn't fix the problem in my case :(

**Comment by supermuesli** [comment]
pull the changes and try again

File: packages/youtube_player_iframe/lib/src/helpers/player_fragments.dart [hunk 1]
```diff
@@ -36,7 +36,7 @@ String youtubeIFrameTag(YoutubePlayerController controller) {
   return '<iframe id="player" type="text/html"'
       ' style="position:absolute; top:0px; left:0px; bottom:0px; right:10px;'
       ' width:100%; height:100%; border:none; margin:0; padding:0; overflow:hidden; z-index:999999;"'
-      ' src="$sourceUri" frameborder="0" allowfullscreen></iframe>';
+      ' src="$sourceUri" frameborder="0" allow="autoplay;fullscreen" allowfullscreen></iframe>';
```
Comment (sarbagyastha) [inline packages/youtube_player_iframe/lib/src/helpers/player_fragments.dart:39]: We should be able to remove `allowfullscreen`.

**Review by sarbagyastha — CHANGES_REQUESTED** [review]

File: packages/youtube_player_iframe/lib/src/helpers/player_fragments.dart (hunk 1 above)
Comment (RamSakamuri) [reply packages/youtube_player_iframe/lib/src/helpers/player_fragments.dart:39]: @supermuesli @sarbagyastha Can  you please merge this changes it its works

**Comment by sarbagyastha** [comment]
Thank you for your contribution. 
Since, the way how iframe was created has changed. I had to add the change manually.

**Pull request closed without merging**